src/axom/inlet/tests/inlet_Inlet.cpp

```diff
@@ -58,22 +59,17 @@ TEST(inlet_Inlet_basic, getTopLevelBools)
   //
 
   bool value = false;
-  bool found = false;
 
   // Check for existing fields
-  found = inlet->get("foo", value);
-  EXPECT_TRUE(found);
+  value = (*inlet)["foo"];
```

Comment:
Suggestion: It might be good to use both the `operator[]()` and the `get` option in this test, to demonstrate that both are valid.